ToolWrapperLayer/WrapperUtility.cs

```diff
@@ -107,7 +107,7 @@ public static string GetAnalysisScriptPath(string analysisDirectory, string scri
         /// <returns></returns>
         public static string GetInstallationScriptPath(string spritzDirectory, string scriptName)
         {
-            return Path.Combine(spritzDirectory, "Scripts", "InstallScripts", scriptName);
+            return Path.Combine(spritzDirectory, "Scripts", "installScripts", scriptName);
```

Comment:
Note to self: this was a temporary fix, check that it gets changed back appropriately